app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,13 +1,33 @@
 import type { ChangeEvent } from 'react';
+import { useContext } from 'react';
+import { t } from '@18f/identity-i18n';
+import { Button } from '@18f/identity-components';
+import { FormStepsButton, useHistoryParam } from '@18f/identity-form-steps';
 import { PasswordToggle } from '@18f/identity-password-toggle';
-import { FormStepsButton } from '@18f/identity-form-steps';
 import { Alert } from '@18f/identity-components';
 import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import { ForgotPassword } from './forgot-password';
+import { VerifyFlowContext } from '@18f//identity-verify-flow';
 import type { VerifyFlowValues } from '../../verify-flow';
 
 interface PasswordConfirmStepStepProps extends FormStepComponentProps<VerifyFlowValues> {}
 
 function PasswordConfirmStep({ errors, registerField, onChange }: PasswordConfirmStepStepProps) {
+  const { basePath } = useContext(VerifyFlowContext);
+  const [path, setPath] = useHistoryParam(basePath);
```

Comment:
`basePath` is an option of the second parameter. The first parameter is an initial URL, which is optional.

```suggestion
  const [path, setPath] = useHistoryParam(undefined, { basePath });
```

Reply:
On second glance, `basePath` would be the base path for the whole flow (`/verify/v2`), whereas the way we're using it here, we want paths to be appended with the step (`/verify/v2/password_confirm`)

```suggestion
  const [path, setPath] = useHistoryParam(undefined, { basePath: `${basePath}/password_confirm` });
```